mmil.raw.html

```diff
@@ -4417,7 +4417,12 @@
 <TR>
   <TD>findcard3</TD>
   <TD><I>none</I></TD>
-  <TD>The set.mm proof is in terms of strict dominance.</TD>
+  <TD>Compare with ~ findcard2sd in which ` y` is finite and is
```

Comment:
`` ` y` `` =>  `` ` y ` ``

Reply:
Hmm, I thought there was an automated check for that but I guess not.

Anyway, fixed.